checks/doks/admission_controller_webhook_timeout.go

```diff
@@ -49,35 +49,55 @@ func (w *webhookTimeoutCheck) Run(objects *kube.Objects) ([]checks.Diagnostic, e
 
 	for _, config := range objects.ValidatingWebhookConfigurations.Items {
 		for _, wh := range config.Webhooks {
-			if *wh.TimeoutSeconds >= int32(1) && *wh.TimeoutSeconds < int32(30) {
-				// Webhooks with TimeoutSeconds set: between 1 and 30 is fine.
+			if wh.TimeoutSeconds == nil {
+				// TimeoutSeconds value should be set to a non-nil value (greater than or equal to 1 and less than 30).
+				d := checks.Diagnostic{
```

Comment:
If a cluster's version is >1.14, then the api server adds the default 30s if timeoutSeconds is not specified. If the cluster version is 1.13.*, users cannot configure the timeoutSeconds and this will be left as nil. 

Now that we have figured out when the timeoutSeconds is nil and users can't do anything about it if they are on 1.13 or lower, I don't think we should be adding this diagnostic. We should simply continue

Reply:
Great find Varsha.

Not sure what the default behavior with regards to web hooks not responding back in 1.13 was, but with no action possible I agree that skipping such web hooks is the best thing to do. (Perhaps we could just document our finding as a code comment.)

Reply:
Just saw these comments @varshavaradarajan @timoreimann, just made the update. Thank you!